client/client-multi/client-cpp/Modules/FindBotan.cmake

```diff
@@ -30,37 +33,46 @@ if(NOT WIN32)
 endif(NOT WIN32)
 
 # find the include files
-find_path(BOTAN_INCLUDE_DIR botan/version.h
+find_path(BOTAN_INCLUDE_DIR
+    botan/version.h
     HINTS
     ${CMAKE_FIND_ROOT_PATH}/include
-    ${BOTAN_INCLUDE_DIRS})
+    ${BOTAN_INCLUDE_DIRS}
+    ${CMAKE_INCLUDE_PATH})
```

Comment:
What do you want to do with it? Sorry, but there I just copy-paste FindAvro.cmake